pubpub/pubpub
# Fix: linkable, referenceable math nodes

Restores linking functionality to display math nodes.
Addresses one of the bugs identified in #2137.

_Test plan_
+ go to a [pub with a display math node](http://localhost:9876/pub/m9xhcfz8/draft)
+ copy the link to that node
+ paste the link in new tab
+ should be navigated to that math node
